package.json

```diff
@@ -23,6 +23,9 @@
   "files": [
     "dist"
   ],
+  "peerDependencies": {
+    "@floating-ui/dom": "1.6.3"
+  },
```

Comment:
If @floating-ui/dom is a peerDependency, do we still need to have it under devDependencies?

Reply:
Not sure! @imprashast Do you know if we should have `@floating-ui/dom` both as a peerDependency and a devDependency?

Reply:
Since we are not building anything in this app, we don't need it under devDep but just peerDep :)

Reply:
Updated in this commit: [ea3c6c1](https://github.com/warp-ds/core/pull/15/commits/ea3c6c142f3a98523b57389bf5bb8bb2d76aa3b3)